### PR TITLE
Feature/miner actor face

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
### Description of the Change
Miner actor interface.

### CODE REVIEW

**Review by turuslan — APPROVED** [review]

File: core/vm/actor/builtin/v2/miner/miner_actor.cpp [hunk 1]
```diff
@@ -12,4 +12,45 @@ namespace fc::vm::actor::builtin::v2::miner {
     return outcome::success();
   }
 
+  ACTOR_METHOD_IMPL(ConfirmUpdateWorkerKey) {
+    // TODO (a.chernyshov) implement
+    return outcome::success();
```
Comment (wer1st) [inline core/vm/actor/builtin/v2/miner/miner_actor.cpp:17]: Why not kNotImplemented?

File: core/vm/actor/builtin/v2/miner/miner_actor.hpp [hunk 2]
```diff
@@ -11,8 +11,18 @@ namespace fc::vm::actor::builtin::v2::miner {
...
+  using ProveCommitSector = v0::miner::ProveCommitSector;
+  using ExtendSectorExpiration = v0::miner::ExtendSectorExpiration;
+  using TerminateSectors = v0::miner::TerminateSectors;
+  using DeclareFaults = v0::miner::DeclareFaults;
+  using DeclareFaultsRecovered = v0::miner::DeclareFaultsRecovered;
   using OnDeferredCronEvent = v0::miner::OnDeferredCronEvent;
-  using ConfirmSectorProofsValid = v0::miner::ConfirmSectorProofsValid;
+  using CheckSectorProven = v0::miner::CheckSectorProven;
 
   struct ApplyRewards : ActorMethodBase<14> {
```
Comment (wer1st) [inline core/vm/actor/builtin/v2/miner/miner_actor.hpp:27]: Do we have no AddLockedFund method in v2?

Comment (Alexey-N-Chernyshov) [reply core/vm/actor/builtin/v2/miner/miner_actor.hpp:27]: No, it was renamed to `ApplyRewards`.

File: core/vm/actor/builtin/v2/miner/miner_actor.cpp (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply core/vm/actor/builtin/v2/miner/miner_actor.cpp]: fixed

File: core/vm/actor/builtin/v2/miner/miner_actor.hpp (hunk 2 above)
Comment (wer1st) [reply core/vm/actor/builtin/v2/miner/miner_actor.hpp:27]: ok

**Review by wer1st — APPROVED** [review]

**Pull request merged**